database/src/main/resources/db/migration/V1.50.0__NOTIF-603_add_org_id.sql

```diff
@@ -0,0 +1,9 @@
+ALTER TABLE email_aggregation
+  ADD COLUMN org_id text;
+
+CREATE INDEX email_aggregation_idx ON public.email_aggregation (org_id);
+
+ALTER TABLE event
+  ADD COLUMN org_id text;
+
+CREATE INDEX event_idx ON public.event (org_id);
```

Comment:
And `ix_event_org_id`

Reply:
Also will the other tables, indexes and constraints used on/with `account_id` part of a separate PR ? (let's make sure we don't bring performance regressions)

Reply:
yes, I would like to remove that later with the step/phase 2. The goal if this PR is just add the orgId, collect and observe it for a while and make sure nothing breaks the existing accountId use cases.